benchmarks/grpo/evaluate/gsm8k_eval.py

```
@@ -0,0 +1,42 @@
+def extract_boxed(text: str) -> str:
+    match = re.search(r"\\boxed\{(.+?)\}", text)
+    return match.group(1).strip() if match else ""
+
+def evaluate(predictions_path: str, eval_data_path: str) -> dict:
+    preds = [json.loads(l) for l in open(predictions_path)]
+    evals = [json.loads(l) for l in open(eval_data_path)]
+
+    correct = 0
+    total = min(len(preds), len(evals))
+
+    for pred, ev in zip(preds[:total], evals[:total]):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)